lib/graphql/schema.graphql

```diff
@@ -2452,7 +2452,7 @@ type OrderType {
   description: String
 
   """
-  Custom user message to show with the order, e.g. a special dedication, "in
+  Custom user message to show with the order, e.g., a special dedication, "in
```

Comment:
`schema.graphql` should not be updated manually, only through `npm run graphql:update`. Is there a matching API PR?

Reply:
@Betree : It's a mistake, I think we shouldn't change this file as this is not a user facing change. I've reverted this.